pkg/pgmodel/pgx.go

```diff
@@ -686,6 +685,7 @@ func decompressChunks(conn pgxConn, pending *pendingBuffer, table string) error
 	if delayBy > maxDelayBy {
 		delayBy = maxDelayBy
 	}
+	log.Warn("msg", fmt.Sprintf("Table %s was compressed, decompressing", table), "table", table, "min-time", minTime, "age", time.Since(minTime), "delay-job-by", delayBy)
```

Comment:
nit:could we add a metric as well so we can trend of Grafana

Reply:
let's do that in another PR